extra/release_tool.py

```diff
@@ -1887,6 +1887,8 @@ def do_integration_versions_including(args):
                  "--sort=-version:refname:short",
                  "refs/tags/*",
                  "refs/remotes/%s/master" % remote,
+                 "refs/remotes/%s/staging" % remote,
+                 "refs/remotes/%s/feature/*" % remote,
```

Comment:
I *think* this may cause our integration-test-runner to build all feature branches that contain master, which we probably don't want. We may need to find a better way to handle this.

Reply:
Okey. The `feature/*` part is not in use yet, so I will verify your thinking and just remove it until I find a better solution.

As for the staging part: here it won't be a problem as integration/staging will not be using any component in its master version.

Thanks!